test/functional/ministerial_roles_controller_test.rb

```diff
@@ -180,33 +180,6 @@ class MinisterialRolesControllerTest < ActionController::TestCase
     assert_equal whips, assigns(:whips_by_organisation)
   end
 
-  view_test "shows the cabinet minister's name and role" do
```

Comment:
These tests are probably still needed, since Whitehall still renders these pages.

Reply:
Added back in :)